### DISCUSSION TITLE
Lexical DevTools

### DISCUSSION THREAD

**Discussion by nimeshnayaju** [facebook/lexical#6350] [Show and tell]
Hi, I just published the Lexical DevTools browser extension that lets you inspect Lexical editor state. It supports viewing your editor state as a tree representation (similar to React DevTools), highlighting document node on hover of the tree node, selection panel, and more! It is available for download in [Chrome](https://chromewebstore.google.com/detail/lexical-devtools/dmbopeepjkdlplkjcjbnfiikajiddhnd), and the full source code is available on [GitHub](https://github.com/nimeshnayaju/lexical-devtools).

  <kbd>
    <picture>
      <source media="(prefers-color-scheme: dark)" srcset="https://github.com/nimeshnayaju/lexical-devtools/assets/35807215/4acd42f3-362b-4523-988a-fea9914e9af9">
      <img alt="Screenshot of Lexical DevTools" src="https://github.com/nimeshnayaju/lexical-devtools/assets/35807215/f79c7551-8f98-42cc-b11f-c37788225dc0">
    </picture>
  </kbd>

- Chrome: https://chromewebstore.google.com/detail/lexical-devtools/dmbopeepjkdlplkjcjbnfiikajiddhnd

- GitHub: https://github.com/nimeshnayaju/lexical-devtools

**Comment by jannikbuschke** [comment]
nice 🙏
